src/modules/redesign/parsers/vehicle.ts

```diff
@@ -397,7 +397,7 @@ export default <RedesignParser<VehicleWindow>>(({
         own_cells,
         alliance_cells,
         prisoners_releaseable: !!doc.querySelector(
-            'a[href$="/gefangene/entlassen"]'
+            'a[href^="/missions/"][href*="/gefangene/entlassen?vehicle_id="]'
```

Comment:
afaik is the parameter 'vehicle_id' not necessary

Reply:
it is (otherwise a Serverfehler will appear on this vehicle until it changes its status). Ingame the vehicle_id is always given on that link.